File: frontend/src/constants/issue-type.ts
// ---------------------------------------------
// Issue Type Constants
// Phân loại sự cố phát sinh tại bãi xe
// ---------------------------------------------

export enum IssueType {
  LOST_TICKET = "LOST_TICKET",
  WRONG_PLATE = "WRONG_PLATE",
  OVERTIME = "OVERTIME",
  WRONG_PLACE = "WRONG_PLACE",
  UNPAID_EXIT = "UNPAID_EXIT",
}

export const ISSUE_TYPE_LABELS: Record<IssueType, string> = {
  [IssueType.LOST_TICKET]: "Mất vé/thẻ xe",
  [IssueType.WRONG_PLATE]: "Sai biển số",
  [IssueType.OVERTIME]: "Quá giờ gửi",
  [IssueType.WRONG_PLACE]: "Gửi sai khu vực",
  [IssueType.UNPAID_EXIT]: "Ra bãi chưa thanh toán",
};

File: backend/src/main/java/vn/edu/ut/pbms/constant/IssueType.java
package vn.edu.ut.pbms.constant;

/**
 * Loại sự cố trong Feedback.
 * LOST_TICKET = mất thẻ, WRONG_PLATE = sai biển số,
 * OVERTIME = quá giờ, WRONG_PLACE = đỗ sai chỗ, UNPAID_EXIT = ra bãi không thanh toán.
 */
public enum IssueType {
    LOST_TICKET,
    WRONG_PLATE,
    OVERTIME,
    WRONG_PLACE,
    UNPAID_EXIT
}
